### DISCUSSION TITLE
Check AccessState for Bluetooth

### DISCUSSION THREAD

**Discussion by tomasmcguinness** [shinyorg/shiny#1089] [Q&A] [answered]
I'm using the Shiny alpha in a MAUI application.

I want to check the state of permissions without requesting them, so I can tailor my UI. 

The RequestAccess() call on the BleManager will prompt the user to grant permission if they have never been asked before. Is there anyway to check the AccessState without requesting the permission?

I found an extension method called GetAccessState(), but that seems to be for Blazor. Is there a way to check permissions without requesting them?

Thanks.

**Comment by aritchie** [comment, marked as answer]
No.  It is a request method only.
